apps/condo/domains/organization/integrations/sbbol/SbbolSecretStorage.js

```diff
@@ -38,48 +38,61 @@ class SbbolSecretStorage {
         await this.#setValue('clientSecret:updatedAt', dayjs().toISOString())
     }
 
-    async getAccessToken () {
-        return this.#getValue('accessToken')
+    async getAccessToken (userId) {
+        return this.#getValue(`${userId}:accessToken`)
     }
 
-    async setAccessToken (value, options) {
-        await this.#setValue('accessToken', value, options)
-        await this.#setValue('accessToken:updatedAt', dayjs().toISOString())
+    /**
+     * Returns an object where the key is a copy of the key stored in the redis and the value is the corresponding accessToken
+     * @returns {Promise<{redisKey: accessToken}[]>}
+     */
+    async getAllAccessToken () {
```

Comment:
Getting all access tokens from Redis just for one value seems to be ineffective from performance point of view. What if we will have thousands of users. Will some limits be violated?
Also, the method used only in one place and it seems like there is no other use-cases.
We have a `service_organization_hashOrgId` environment variable, you can get appropriate `Organization` record and first `User` imported from SBBOL. This way we will pass `accessToken` belonging to Organization, corresponding to our integration. Technically it does not matters for SBBOL, but it will help to avoid confusion about this strange sampling.